File: src/Domain/Entities/Games/Game.cs
using Domain.Enums;

namespace Domain.Entities.Games;

public class Game : EntityWithId
{
    public string Code { get; set; }

    public string Name { get; set; }

    public GameStatus Status { get; set; }

    public bool IsPrivate { get; set; }

    public string GameMasterUserId { get; set; }

    public List<string> UserIds { get; set; }

    public List<Round> Rounds { get; set; }
}

File: src/react-frontend/src/app/types/models/game/game.ts
import { GameStatus } from 'app/types/enums/gameStatus';
import { UserDto } from '../user/userDto';
import { Round } from './round';

export type Game = {
  id: string;
  code: string;
  name: string;
  status: GameStatus;
  isPrivate: boolean;
  gameMasterUser: UserDto;
  users: UserDto[];
  currentRound?: Round;
  previousRounds: Round[];
};
